src/Commands/Images/GenerateAbstract.php

```diff
@@ -0,0 +1,376 @@
+<?php
+/**
+ * 2019-present Friends of Presta community
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the MIT License
+ * that is bundled with this package in the file LICENSE
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/MIT
+ *
+ * @author Friends of Presta community
+ * @copyright 2019-present Friends of Presta community
+ * @license https://opensource.org/licenses/MIT MIT
+ */
+
+namespace FOP\Console\Commands\Images;
+
+use Configuration;
+use FOP\Console\Command;
+use Image;
+use ImageManager;
+use ImageType;
+use Language;
+use Module;
```

Comment:
I guess these error are linked with your IDE, as all the classes are well used in this command
Maybe it doesn't index the class_stub.php file ?
egrep "ImageManager|ImageType|Language|Module" src/Commands/Images/GenerateAbstract.php 
` use ImageManager;
use ImageType;
use Language;
use Module;
        $languages = Language::getLanguages(false);
            $formats = ImageType::getImagesTypes($proc['type']);
            $formated_medium = ImageType::getFormattedName('medium');
                            } elseif (!ImageManager::resize($dir . $image, $newDir . substr(str_replace('_thumb.', '.', $image), 0, -4) . '-' . stripslashes($imageType['name']) . '.jpg', (int) $imageType['width'], (int) $imageType['height'])) {
                                if (!ImageManager::resize($dir . $image, $newDir . substr($image, 0, -4) . '-' . stripslashes($imageType['name']) . '2x.jpg', (int) $imageType['width'] * 2, (int) $imageType['height'] * 2)) {
                            if (!ImageManager::resize($existing_img, $dir . $imageObj->getExistingImgPath() . '-' . stripslashes($imageType['name']) . '.jpg', (int) $imageType['width'], (int) $imageType['height'])) {
                                if (!ImageManager::resize($existing_img, $dir . $imageObj->getExistingImgPath() . '-' . stripslashes($imageType['name']) . '2x.jpg', (int) $imageType['width'] * 2, (int) $imageType['height'] * 2)) {
                    $file = _PS_PROD_IMG_DIR_ . Language::getIsoById((int) Configuration::get('PS_LANG_DEFAULT')) . '.jpg';
                    if (!ImageManager::resize($file, $dir . $language['iso_code'] . '-default-' . stripslashes($image_type['name']) . '.jpg', (int) $image_type['width'], (int) $image_type['height'])) {
                        if (!ImageManager::resize($file, $dir . $language['iso_code'] . '-default-' . stripslashes($image_type['name']) . '2x.jpg', (int) $image_type['width'] * 2, (int) $image_type['height'] * 2)) {
                        $moduleInstance = Module::getInstanceByName($module['name']);
`

Reply:
ha ye my bad didn't see the static call to a Module method line 367 (i'm reviewing on github but ye if i was on my editor it would be simplier :D )

Then it's a perfect example of usage of using `\Module` instead of adding a use at the top of the file 😛